src/plugin/transform.ts

```diff
@@ -290,13 +331,17 @@ export function transform({
                 }
 
                 const tabsGroup = match[2] || `${DEFAULT_TABS_GROUP_PREFIX}${generateID()}`;
+                const orientation = (match[4] || 'horizontal') as TabsOrientation;
 
                 const {tabs, index} = findTabs(state.tokens, i + 3);
 
+                debugger;
```

Comment:
I received a notification about PR. Is it ready to review?

Reply:
No, I just pushed it to check by myself, sry for notification

Reply:
Okay. You can call me when it will be ready to review. I'll try to make it fast